### PR TITLE
versions: refactor configuration of qemu in version.yaml

### PR DESCRIPTION
**Pull request by teawater**
Include info such as commit number,tag number, etc for building
nvdimm-supported qemu on aarch64 from source code.

Fixes: #974

Signed-off-by: Hui Zhu <teawater@hyper.sh>

### CODE REVIEW

File: versions.yaml [hunk 1]
```diff
@@ -69,8 +69,12 @@ assets:
     qemu:
       description: "VMM that uses KVM"
       url: "https://github.com/qemu/qemu"
-      version: "stable-2.11"
-
+      version: "stable-2.12"
```
Comment (jodh-intel) [inline versions.yaml]: Is this change required? If so, please put it into a separate commit explaining why it's changing.

**Comment by teawater** [comment]
jenkins-metrics-ubuntu-16-04 failed in:
```
12:10:31 +-----+----------------------+-------+--------+--------+-------+--------+--------+-------+------+-----+
12:10:31 | P/F |         NAME         |  FLR  |  MEAN  |  CEIL  |  GAP  |  MIN   |  MAX   |  RNG  | COV  | ITS |
12:10:31 +-----+----------------------+-------+--------+--------+-------+--------+--------+-------+------+-----+
12:10:31 | *F* | boot-times           | 95.0% | 107.4% | 105.0% | 10.0% | 104.7% | 121.1% | 15.6% | 3.2% |  20 |
12:10:31 | *F* | memory-footprint     | 85.0% | 84.9%  | 115.0% | 30.0% | 84.9%  | 84.9%  | 0.0%  | 0.0% |   1 |
12:10:31 | P   | memory-footprint-ksm | 95.0% | 100.6% | 105.0% | 10.0% | 100.6% | 100.6% | 0.0%  | 0.0% |   1 |
12:10:31 +-----+----------------------+-------+--------+--------+-------+--------+--------+-------+------+-----+
```
But this patch doesn't have relate with this fail.  Maybe it can be ignored.

**Comment by bergwolf** [comment]
LGTM. metrics is still failing but not related to the PR.

**Comment by amshinde** [comment]
Ignoring the metrics failure as it is not related.
